client/src/components/application/DotsStepper/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import MobileStepper from '@material-ui/core/MobileStepper';
+import PropTypes from 'prop-types';
+import dotstyle from './style';
+import regestration from './regestrationSteps.json';
```

Comment:
here is a typo (regestration) instead of registration ,also the same in line 10.
you can install plugin in vs-code to check error typo like [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)

Reply:
thanks, @mFlaifel  for this extension